### DISCUSSION TITLE
Individual arrow keys on keyboard instead of combined arrow key

### DISCUSSION THREAD

**Discussion by chrisbennight** [blinksh/blink#2011] [Support] [answered]
I recently swapped to a new ipad and the arrow keys are now a combined
![IMG_0001](https://github.com/blinksh/blink/assets/1864782/18958799-a28c-41bb-9169-d9e354e2d8da)
Which i find pretty frustrating to use.   Navigating a precise number of presses in any direction is near impossible.

I have not been able to find the setting to get the standard individual arrow keys back
![IMG_0002](https://github.com/blinksh/blink/assets/1864782/df5da79e-c1a8-49be-9f2f-b1118a3aa466)

**Comment by chrisbennight** [comment]
This is on a m4 13" ipad if it is relevant

**Comment by chrisbennight** [comment]
Would appreciate some input from the team clarifying if this is this a "new feature" or a regression.  Vim is borderline unusable with this, so going to have to make a call if I need to migrate to another app or not

**Reply by carloscabanero** [reply, marked as answer]
Hi thanks for the heads up. This is related to #2009 . Will try to provide a hot fix before 17.3.0.

**Reply by chrisbennight** [reply]
I can confirm this is fixed in 17.3.0, thanks

**Comment by Lyapsus** [comment]
Honestly would be cool to have it as an option on iPhones too
